Dockerfile

```diff
@@ -19,5 +19,7 @@ ENV PYTHONUNBUFFERED 1
 
 ARG IMAGE_TAG
 ARG IMAGE_REPOSITORY
+ARG DATA_DIR
```

Comment:
Let's not make data_dir configurable. This is what the container sees, and does not need to be changed.

Reply:
Ah okay, I'll undo this change